bin/main

```diff
@@ -1,65 +1,48 @@
 #!/usr/bin/env ruby
+# frozen_string_literal: true
+
+require './lib/logic.rb'
+
 class TicTacInterface
-  def initialize()
+  def initialize
+    @engine = TicTacLogic.new
     begin
       puts `clear`
       p "What's your name, player 1?"
-      @player1 = gets.chomp
+      @engine.player1 = gets.chomp
       puts `clear`
       p "What's your name, player 2?"
-      @player2 = gets.chomp
+      @engine.player2 = gets.chomp
       puts `clear`
     rescue StandardError => e
-      p "No name given"
+      p "No name given #{e}"
+      sleep 1
       retry
-    else
     end
```

Comment:
According to that, it's possible for you to initialize the TicTacLogic instance with players as parameters.
Why not do that ?